WinUIGallery/ControlPages/DesignGuidance/IconsPage.xaml

```diff
@@ -183,18 +163,19 @@
                     HorizontalAlignment="Stretch"
                     VerticalAlignment="Stretch"
                     AutomationProperties.Name="Icons">
-                    <ItemsRepeater
-                        x:Name="IconsRepeater"
+                    <ItemsView
+                        x:Name="IconsItemsView"
                         MinWidth="100"
                         Margin="0,0,0,36"
                         HorizontalAlignment="Stretch"
-                        ElementIndexChanged="IconsRepeater_ElementIndexChanged"
-                        ElementPrepared="IconsRepeater_ElementPrepared"
-                        ItemTemplate="{StaticResource IconTemplate}">
-                        <ItemsRepeater.Layout>
+                        IsItemInvokedEnabled="True"
```

Comment:
I think you can leave this to the default False value. You don't listen to the Invoked event.